samples/README.md

```diff
@@ -0,0 +1,19 @@
+## Overview
+This folder contains some end-to-end samples using the pvsadm tool
+
+### 1. Image conversion and upload to pvs instance sample script
+The image_create_and_upload_script helps to create ova image from qcow2 image, upload the images to a bucket (new/existing) and import them as boot image to pvs instance using the pvsadm tool
+#### Prerequisite
+1. Access to shell on IBM Power® logical partition (LPAR) running RHEL 8.xor CentOS 8.x with internet connectivity and minimum 250GB of free disk space
```

Comment:
extremely sorry.. missed this "RHEL 8.xor CentOS 8.x". This should be "RHEL 8.x or CentOS 8.x"

Reply:
rest lgtm

Reply:
Changed it